### PR TITLE
changed img storage location

### PR DESCRIPTION
**Pull request by TrevorOctober**
@hodgestar Ready for review cc @KaitCrawford

### CODE REVIEW

**Comment by TrevorOctober** [comment]
@hodgestar Ready for re-review

File: mentorship/mentor/models.py [hunk 1]
```diff
@@ -25,9 +26,11 @@ class Mentee(Contact):
 
 
 class Mentor(Contact):
+    profile_uploads = settings.PROFILE_UPLOADS
     mentee = models.OneToOneField(Mentee, related_name="mentee", null=True,
                                   blank=True)
-    profile_pic = models.ImageField(upload_to="media/", null=True, blank=True)
+    profile_pic = models.ImageField(
+        upload_to=profile_uploads, null=True, blank=True)
```
Comment (hodgestar) [inline mentorship/mentor/models.py]: Just use `upload_to=settings.PROFILE_UPLOADS` directly here (we probably don't want the setting available on the class).

**Comment by hodgestar** [comment]
One small comment, otherwise looks good.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**